### PR TITLE
[CoreLocation] Updates for Xcode 14.0.0

### PR DESCRIPTION
**Pull request by tj-devel709**

### CODE REVIEW

File: src/corelocation.cs [hunk 1]
```diff
@@ -474,6 +474,9 @@ partial interface CLLocationManager {
 		[Export ("stopMonitoringLocationPushes")]
 		void StopMonitoringLocationPushes ();
 
+		[Watch (9,0), NoTV, NoMac, NoiOS, NoMacCatalyst]
+		[Export ("requestHistoricalLocationsWithPurposeKey:sampleCount:completionHandler:")]
+		void RequestHistoricalLocation (string purposeKey, nint sampleCount, Action<NSArray<CLLocation>, NSError> handler);
```
Comment (rolfbjarne) [inline src/corelocation.cs:479]: Looks like this method could be `[Async]`

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

File: src/corelocation.cs (hunk 1 above)
Comment (tj-devel709) [reply src/corelocation.cs:479]: Good Call!

Comment (tj-devel709) [reply src/corelocation.cs:480]: @rolfbjarne addressed!

**Comment by rolfbjarne** [comment]
@tj-devel709 I think you'll have to close this PR and open a new one that targets xcode14.1, because the diff is weird.

**Comment by tj-devel709** [comment]
Closing here, will open up a PR in to the Xcode14.1 branch

**Pull request closed without merging**